library/CHART-TEMPLATE/values.yaml

```diff
@@ -15,6 +15,8 @@ global:
   tolerations: []
   affinity: {}
 
+licenseKey: foobar
```

Comment:
I would not have this enabled by default in the chart example

Reply:
Unfortunately, if we don't do it then the secret will fail to template.

I could get rid of it and implement the `allowEmpty` override though, if you think that's better.

Reply:
somehow I read `customLicenseKey`. No need to change that 😄

Reply:
@kang-makes We can create an unit testing chart with sub-charts for these very corner cases. But I'd vouch for doing that in another PR :)

Reply:
Yeah, in case we have time to do it is going to be done after GA Helm v3 charts...